frontend/src/pages/Members.vue

```diff
@@ -229,7 +239,7 @@ export default {
       serviceAccountHelpDialog: false,
       kubeconfigDialog: false,
       updateMemberName: undefined,
-      updateUserRoles: undefined,
+      updateMemberRoles: undefined,
```

Comment:
It confuses me completely if a property has the name of a method and vice versa.  I expect `updateMemberRoles` and `updateMemberName` to be a method nd not a property. I expect `roleName` to be a property and not a method which returns the displayName for a given role.

Reply:
renamed to update**d**